src/structs/types.ts

```diff
@@ -457,6 +457,14 @@ export function object<Schema extends ObjectSchema>(
 
         for (const key of knowns) {
           unknowns.delete(key);
+          const propertySchema = schema[key] as Struct;
```

Comment:
```suggestion
          const propertySchema = schema[key];
```

No need to cast with the above suggestion.